eradiate/scenes/atmosphere/_particles.py

```diff
@@ -370,18 +371,21 @@ def eval_sigma_t(self, spectral_ctx: SpectralContext) -> pint.Quantity:
     def eval_sigma_t_mono(self, w: pint.Quantity) -> pint.Quantity:
 
         ds = xr.open_dataset(self.dataset)
-        wavelengths = w.m_as(ds.w.attrs["units"])
-        interpolated_sigma_t = ds.sigma_t.interp(w=wavelengths)
-        sigma_t = to_quantity(interpolated_sigma_t)
+        ds_w_units = ds.w.attrs["units"]
+        wavelength = w.m_as(ds_w_units)
+        sigma_t = to_quantity(ds.sigma_t.interp(w=wavelength))
+        sigma_t_550 = to_quantity(
+            ds.sigma_t.interp(w=ureg.Quantity(550.0, "nm").m_as(ds_w_units))
```

Comment:
For optimal performance, I'd rather rather go with
```suggestion
            ds.sigma_t.interp(w=ureg.convert(550.0, ureg.nm, ds_w_units))
```